cpanfile

```diff
@@ -43,6 +43,7 @@ requires 'Email::Sender'                              => '1.300010';
 requires 'Email::Valid'                               => '1.192';
 requires 'Encode::Detect'                             => '1.01';
 requires 'Exception::Class'                           => '1.37';
+requires 'File::Slurp';
```

Comment:
For production systems, this is installed as a dependency of `Catalyst::Plugin::ErrorCatcher`, but it is required on other systems, too.